pcdsdevices/epics_motor.py

```diff
@@ -152,6 +154,50 @@ def disable(self):
         """
         return self.disabled.set(value=1)
 
+    def stop(self):
+        """
+        Stops the motor.
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
+        """
+        return self.motor_spg.put(value='Stop')
+
+    def pause(self):
+        """
+        Pauses a move.
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
```

Comment:
This doc string (and the others) are incorrect. `put` does not return a `Status` object, it returns `None`.

Reply:
I think this docstring is the only complaint I have left